packages/cfd/src/Components/__tests__/success.dialog.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import SuccessDialog from '../success-dialog';
+import userEvent from '@testing-library/user-event';
+
+type TModal = React.FC<{
+    children: React.ReactNode;
+    is_open: boolean;
+    has_close_icon: boolean;
+    toggleModal: () => void;
+    title: string;
+}> & {
+    Body?: React.FC<{
+        children: React.ReactNode;
+    }>;
+    Footer?: React.FC<{
+        children: React.ReactNode;
+    }>;
+};
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    const Icon = jest.fn(() => <div>Checkmark</div>);
+    const Modal: TModal = jest.fn(({ children, is_open, has_close_icon, toggleModal, title }) => {
+        if (is_open)
+            return (
+                <div data-testid='modal'>
+                    <h3>{title}</h3>
+                    {has_close_icon && <span onClick={toggleModal}>IcCross</span>}
+                    {children}
+                </div>
+            );
+
+        return null;
+    });
+    Modal.Body = jest.fn(({ children }) => <div>{children}</div>);
+    Modal.Footer = jest.fn(({ children }) => <div>{children}</div>);
+
+    return {
+        ...original_module,
+        Modal,
+        Icon,
+    };
+});
+
+describe('<SuccessDialog />', () => {
+    const modal_root_el = document.createElement('div');
+    modal_root_el.setAttribute('id', 'modal_root');
+    document.body.appendChild(modal_root_el);
+    const mock_message = 'mockMessage';
+
+    let props: React.ComponentProps<typeof SuccessDialog>;
+    beforeEach(() => {
+        props = {
+            classNameMessage: 'classNameMessage',
+            has_cancel: false,
```

Comment:
Those props (`has_cancel` and `has_submit`) have default properties in lines 32-33 (success-dialog.tsx), which are also `false` for `has_cancel` and `true` for `has_submit`. So, maybe there is no need in passing default values? Just asking

Reply:
yeah they have default value but still added in the mock props for better readability.
Can Remove them, but i think its fine